src/nifake/tests/test_session.py

```diff
@@ -534,53 +535,30 @@ def test_read_with_warning(self):
                 assert test_error_desc in str(w[0].message)
     '''
 
-    def test_enum_input_function_with_defaults(self):
-        test_turtle = nifake.Turtle.DONATELLO
-        self.patched_library.niFake_EnumInputFunctionWithDefaults.side_effect = self.side_effects_helper.niFake_EnumInputFunctionWithDefaults
-        with nifake.Session('dev1') as session:
-            session.enum_input_function_with_defaults()
-            session.enum_input_function_with_defaults(test_turtle)
-            from mock import call
-            calls = [call(SESSION_NUM_FOR_TEST, 0), call(SESSION_NUM_FOR_TEST, 1)]  # 0 is the value of the default of nifake.Turtle.LEONARDO, 1 is the value of nifake.Turtle.DONATELLO
-            self.patched_library.niFake_EnumInputFunctionWithDefaults.assert_has_calls(calls)
+    # TODO(marcoskirsch):
```

Comment:
I had the TODOs located where if replaced with a test, the tests would be in a more or less logical order. I don't think test order should be chronological (when it was written). Ideally they follow some logical order. Will developers know to do this?